<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>widget_combo</class>
 <widget class="QWidget" name="widget_combo">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>48</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>640</width>
    <height>48</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QComboBox" name="cbxChoices">
   <property name="geometry">
    <rect>
     <x>486</x>
     <y>10</y>
     <width>141</width>
     <height>32</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="lblCombo">
   <property name="geometry">
    <rect>
     <x>32</x>
     <y>16</y>
     <width>351</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Text</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
